estela-web/src/components/Stats/ChartsSection.tsx

```diff
@@ -187,11 +187,11 @@ const getLogsDataset = (statData: GlobalStats[]) => {
 
 interface ChartsSectionProps {
     loadedStats: boolean;
-    stats: GlobalStats[] | SpidersJobsStats[];
+    stats: ProjectStats[] | SpidersStats[];
```

Comment:
I have found another instance of `SpidersJobsStats` in `src/components/Stats/StatsTableSection.tsx` in case that should also be replaced. Also, the `SpidersJobsStats` class is still in the autogenerated API. Is that right?

Reply:
I'll replace `StatsTableSection.tsx` file with other ticket by now since the expandable functionality is disabled this imports don't affect anything in the frontend.

Reply:
Anyway I changed the models by the new ones.